site/content/how-to/monitoring/tracing.md

```diff
@@ -0,0 +1,335 @@
+
+## Installing the Collectors
+
+The first step is to install the collectors. NGINX Gateway Fabric will be configured to export to the OpenTelemetry Collector, which is configured to export to Jaeger. This model allows us to easily swap out the visualization collector (Jaeger) for something else if we want to, or add more collectors without needing to reconfigure NGINX Gateway Fabric. It is also possible to configure NGINX Gateway Fabric to export directly to Jaeger, if desired.
+
+First, create the namespace:
+
+```shell
+kubectl create namespace monitoring
+```
+
+Download the following files containing the configurations for the collectors:
+
+- {{< download "otel-collector.yaml" "otel-collector.yaml" >}}
+- {{< download "jaeger.yaml" "jaeger.yaml" >}}
+
+{{< note >}}These collectors are for demo purposes and are not tuned for production use.{{< /note >}}
+
+and install:
+
+```shell
+kubectl apply -f otel-collector.yaml -f jaeger.yaml -n monitoring
+```
+
+Ensure that the Pods are running:
+
+```shell
+kubectl -n monitoring get pods
+```
+
+```text
+NAME                             READY   STATUS    RESTARTS   AGE
+jaeger-8469f69b86-bfpk9          1/1     Running   0          9s
+otel-collector-f786b7dfd-h2x9l   1/1     Running   0          9s
+```
+
+Once running, you can access the Jaeger dashboard by using port-forwarding in the background:
+
+```shell
+kubectl port-forward -n monitoring svc/jaeger 16686:16686 &
+```
+
+Visit [http://127.0.0.1:16686](http://127.0.0.1:16686) to view the dashboard.
+
+## Enabling Tracing
+
+Enabling tracing requires two pieces of configuration.
+
+- `NginxProxy`: This resource contains global settings relating to the NGINX data plane. It is created and managed by the [cluster operator](https://gateway-api.sigs.k8s.io/concepts/roles-and-personas/), and is referenced in the `parametersRef` field of the GatewayClass. This resource can be created and linked when we install NGINX Gateway Fabric using its helm chart, or it can be added later. In this guide we will install the resource using the helm chart, but will also show what it looks like in case you want to add it after installation.
+
+The `NginxProxy` resource contains configuration for the collector, and applies to all Gateways and routes under the GatewayClass. It does not enable tracing, but is a prerequisite to the next piece of configuration.
+
+- `ObservabilityPolicy`: This resource is a [Policy](https://gateway-api.sigs.k8s.io/reference/policy-attachment/) that targets HTTPRoutes or GRPCRoutes. It is created by the [application developer](https://gateway-api.sigs.k8s.io/concepts/roles-and-personas/) and enables tracing for a specific route or routes. It requires the `NginxProxy` resource to exist in order to complete the tracing configuration.
+
+TODO(sberman): link to reference docs
```

Comment:
Just calling out this TODO just in case it gets overlooked, though you might have some local tooling that flags it anyway.